tests/vsphere_test.go

```diff
@@ -43,6 +44,24 @@ const (
 
 var templateID string
 
+// This versioning scheme that currently seems to be in place for template build numbers.
+var buildNumberRegex = regexp.MustCompile(`[bB]?(?P<version>\d+)`)
```

Comment:
You declare a subexpression named `version` and some lines below get the index for `build`.

I'd just use this regex: `[bB]?(\d+)` and use `matches[0]` below.

Good job making me learn something about regex (named subexpression) though, that didn't happen in a while :D

Reply:
oh. Forgot that during my last minute refactorings